sweep/sweeper.go

```diff
@@ -1669,6 +1669,22 @@ func (s *UtxoSweeper) handleSweep(bestHeight int32) {
 	}
 }
 
+// IsSweeperOutpoint determines whether the outpoint was created by the sweeper.
+//
+// NOTE: It is enough to check the txid because the sweeper will create
+// outpoints which solely belong to the internal LND wallet.
+func (s *UtxoSweeper) IsSweeperOutpoint(op wire.OutPoint) bool {
+	found, err := s.cfg.Store.IsOurTx(op.Hash)
+	if err != nil {
+		log.Errorf("failed to fetch info for outpoint(%v:%d) with: %v",
+			op.Hash, op.Index, err)
+
+		return false
```

Comment:
Do we want connectivity failure to our data store to result in us believing the output wasn't created by us?

Reply:
Yes this was my initial idea but no strong opinion about it, wanted to create a more simpler function rather than passing down the error message to the caller.

Reply:
perhaps to be safe we should do the opposite? ie, assume the worst right?

Reply:
ok sounds good let's assume the worst.